dev/CommonStyles/TestUI/CompactPage.xaml

```diff
@@ -0,0 +1,53 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<local:TestPage
+    x:Class="MUXControlsTestApp.CompactPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:MUXControlsTestApp"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    RequestedTheme ="Dark"
```

Comment:
Yes. From nuget, I didn't see any problem. but in testapp, not all changes are applied until I change the RequestedTheme

Reply:
Really? That seems like a bug.

Reply:
I think so, but I will not continue the investigation since our testapp is not using MUX as a nuget.
I did the testing manually with nuget and didn't see any problem. 
Problem may because XamlControlsResources is inside a MergedDictionary in App.xaml for the testapp.

Reply:
Please don't just cover this up. We want to understand any limitations of this approach for applying compact and anticipate any problems developers will have in advance. If you're hitting a blocking issue that requires you specify the theme then open an issue to track that.

Reply:
I didn't repro the problem. Now I removed that line, and wait for the testing result